libraries/radpdfprocessing/cross-platform.md

```diff
@@ -77,6 +81,63 @@ The new **FixedExtensibilityManager** class is exposing one property:
         FixedExtensibilityManager.FontsProvider = fontsProvider;
     {{endregion}}
 
+* **JpegImageConverter**: Gets or sets a *JpegImageConverterBase* instance used to provide a Jpeg converted image data.
+
+    >note .NET Standard specification does not define APIs for converting images or scaling their quality. Thats why, to allow the library to export images different than Jpeg and Jpeg2000 or ImageQuality different than High, you will need to provide an implementation of the JpegImageConverterBase abstract class and set this implementation to the JpegImageConverter property of the of FixedExtensibilityManager.
```

Comment:
That is why to allow the library to export images different than Jpeg and Jpeg2000 or ImageQuality different than High, you will need to provide an implementation of the **JpegImageConverterBase** abstract class. This implementation should be passed to the **JpegImageConverter** property of **FixedExtensibilityManager**.